cluster/gce/config-default.sh

```diff
@@ -95,6 +95,21 @@ if [[ "${ENABLE_NODE_AUTOSCALER}" == "true" ]]; then
   TARGET_NODE_UTILIZATION="${KUBE_TARGET_NODE_UTILIZATION:-0.7}"
 fi
 
+# Optional: Create autoscaler for cluster's pods.
+# NOT WORKING YET!
+ENABLE_POD_AUTOSCALER="${KUBE_ENABLE_POD_AUTOSCALER:-false}"
+ENABLE_POD_AUTOSCALER=true
+if [[ "${ENABLE_POD_AUTOSCALER}" == "true" ]]; then
+  if [[ -z "${RUNTIME_CONFIG}" ]]; then
+    RUNTIME_CONFIG="experimental/v1=true"
```

Comment:
How about we have a separate flag that controls experimental API and we just set it to true here. That way we will reuse the same logic for future experimental features. WDYT?